ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Ledger.hs

```diff
@@ -484,27 +486,28 @@ encodeShelleyLedgerState
                        , shelleyLedgerTransition
                        } =
     encodeVersion serialisationFormatVersion2 $ mconcat [
-        CBOR.encodeListLen 2
```

Comment:
This affects the NTC protocol, but only a `Debug*` constructor:

It also affects the on-disk snapshots, but those are already changing, so we're batching this in with that.

```
  -- | Only for debugging purposes, we don't guarantee binary compatibility.
  -- Moreover, it is huge.
  DebugNewEpochState
```